libvast/fbs/value_index.fbs

```diff
@@ -17,7 +17,11 @@ table LegacyValueIndex {
   /// because we can't distinguish `0` from "Field doesn't exist", but that's fine
   /// here because the partition itself will always occupy index 0 of the external
   /// container.
-  external_container_idx: ulong;
+  external_container_idx_caf_0_17: ulong;
```

Comment:
Style nit: All the other adjusted fields had `caf_0_x` as a prefix, this and its replacement have it as a suffix.